### DISCUSSION TITLE
Client not receiving the Socket Info

### DISCUSSION THREAD

**Discussion by R-duterte** [Escovilla/ESP8266-Socket.IO#4] [Q&A] [answered]
this is my script for the client side, the server can receive the data but my client side cannot
```
<script>
      var socket = io('example.com');
      socket.on('message', (temp) => {
        console.log("temp: ",temp )
      });
</script>
```

**Comment by Escovilla** [comment, marked as answer]
check [this ](https://github.com/Escovilla/sampleSocket)out in my example
i added transports: ['websocket'] in my code

var socket = io({transports: ['websocket']});

this is to avoid Cross-site limitations

speaking of

if your client app is in another server then there are Cross-site limitations
this limitation specifies that web pages served from different domains, protocols, or ports cannot access each other's data or interact with each other's scripts, unless they explicitly grant permission to do so.

you can just enable the ip address to acess the server
Access-Control-Allow-Origin: http://example.com/
if you are using nodejs you can add cors
npm i cors
then
const cors = require('cors');
app.use(cors({
    origin: 'https://example.com'
}));
